js/recovery-sdk/src/standardize.ts

```diff
@@ -0,0 +1,41 @@
+import { parsePhoneNumberFromString } from 'libphonenumber-js/min';
+import { ContactType } from './types.js';
+
+/**
+ * Cleans up the email by lowercasing and removing periods in the username
+ * Does NOT validate the email
```

Comment:
Would it be too much lift to validate the email address? We validate the phone number below...

Reply:
How about https://github.com/validatorjs/validator.js ?